components/layout_2020/formatting_contexts.rs

```diff
@@ -219,7 +219,7 @@ impl IndependentFormattingContext {
 }
 
 impl IndependentNonReplacedContents {
-    pub fn layout(
+    pub(crate) fn layout(
```

Comment:
Maybe it would be better to call this `layout_without_caching` and the one with caching just `layout`?

Reply:
Makes sense!